### PR TITLE
chore(mods/magiclysm): Remove myself as Magiclysm Maintainer

### PR DESCRIPTION
**Pull request by RobbieNeko**
## Purpose of change

I'm not actually maintaining or updating Magiclysm anymore.

## Describe the solution

Removes myself from the list of maintainers in the modinfo file, replaces it with "none"

## Describe alternatives you've considered

- PR'ing moving Magical Nights in-repo
Not _**yet**_. But later, perhaps ;3

## Testing

N/A

## Additional context

I'd strongly suggest people interested in updated content that's actually getting maintained check out Magical Nights instead.

### CODE REVIEW

File: data/mods/Magiclysm/modinfo.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "id": "magiclysm",
     "name": "Magiclysm",
     "authors": [ "KorGgenT", "Aptronym", "LaVeyanFiend" ],
-    "maintainers": [ "Robbietheneko" ],
+    "maintainers": [ "none" ],
```
Comment (scarf005) [inline data/mods/Magiclysm/modinfo.json:7]:
```suggestion
    "maintainers": [ ],
```

could it be possible to keep `maintainers` empty (if this work without errors?)

Comment (RobbieNeko) [reply data/mods/Magiclysm/modinfo.json:7]: Very interesting idea, that would probably be better assuming it works without errors yeah.

Comment (RobbieNeko) [reply data/mods/Magiclysm/modinfo.json:7]: I'll explore this option later once I've slept and have the time. Not like this is an urgent change after all, haha.

**Comment by RobbieNeko** [comment]
Funny to see a change suggested by Scarf getting autofixed for once ^-^

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**